crates/bevy_pbr/src/light.rs

```diff
@@ -369,8 +386,12 @@ impl Clusters {
         near: f32,
         far: f32,
     ) -> Self {
+        assert!(screen_size.x > 0 && screen_size.y > 0);
+        assert!(dimensions.x > 0 && dimensions.y > 0 && dimensions.z > 0);
```

Comment:
This is a private method and I think if these are 0, it is our fault, so I think these should be debug_assert too.
```suggestion
        debug_assert!(screen_size.x > 0 && screen_size.y > 0);
        debug_assert!(dimensions.x > 0 && dimensions.y > 0 && dimensions.z > 0);
```